web/src/lib/domain-logic/essentialQuestions.ts

```diff
@@ -71,23 +71,18 @@ export const EssentialQuestions: EssentialQuestion[] = [
   new EssentialQuestion({
     shouldBeResetWhenIndustryChanges: true,
     isQuestionApplicableToIndustry: (industry) => {
-      return (
-        !!industry.industryOnboardingQuestions.isInterstateTransportApplicable && industry.id === "logistics"
-      );
+      return !!industry.industryOnboardingQuestions.isInterstateLogisticsApplicable;
     },
-    fieldName: "interstateTransport",
+    fieldName: "interstateLogistics",
```

Comment:
**Refactor Context:** These were two separate questions that were covered by the same industry onboarding criteria. In deciding how to best approach the refactor, JP and I thought it best to separate out the logistics and moving company industries.